packages/gatsby/src/datastore/in-memory/run-fast-filters.ts

```diff
@@ -444,7 +390,10 @@ function filterToStats(
 function sortNodes(
   nodes: Array<IGatsbyNode>,
   sort:
-    | { fields: Array<string>; order: Array<boolean | "asc" | "desc"> }
+    | {
+        fields: Array<string>
+        order: Array<boolean | "asc" | "desc" | "ASC" | "DESC">
```

Comment:
what prompted addition of `ASC` and `DESC` uppercased variants?

Reply:
Because that's what actually comes from node-module 😑